perceval/backends/core/git.py

```diff
@@ -126,6 +129,7 @@ def fetch_items(self, category, **kwargs):
 
         :returns: a generator of items
         """
+        to_date = kwargs['to_date']
```

Comment:
Put this line below `from_date` to follow the same order you used in the parameters.